x-pack/plugins/canvas/server/sample_data/load_sample_data.ts

```diff
@@ -4,10 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { SampleDataRegistrySetup } from 'src/plugins/home/server';
```

Comment:
Should this be coming off the plugin and passed in here, rather than importing directly?

Reply:
I believe this is just being used for the type. Should I be passing in types via the plugin?